pipelines/tasks/mas/hack-manage-db2.yaml

```diff
@@ -13,6 +13,10 @@ spec:
       type: string
       description: Name (specifically, not the ID) of the DB2 Warehouse instance to execute the hack
       default: "db2wh-db01"
+    - name: db2wh_load_from_config
```

Comment:
This should be removed, it's no longer used in the role (and further below where it's loaded into the stepTemplate too).

Reply:
@alequint would you be able to fix this part when you refactor the pipelines? Don't want to touch in things I know you're working on.

Reply:
Just removed it @durera